app/src/main/java/ca/klapstein/baudit/data/Record.java

```diff
@@ -101,4 +102,9 @@ public GeoLocation getGeoLocation() {
     public void setGeoLocation(GeoLocation geoLocation) {
         this.geoLocation = geoLocation;
     }
+
+    @Override
+    public int compareTo(Record r) {
```

Comment:
Should provide `javadoc` on how this `compareTo` behaves. (i.e. what does it actually compare, what makes the decision onto what is a greater/lesser Record)